acousticbrainz/testing.py

```diff
@@ -21,16 +22,17 @@ def tearDown(self):
         self.truncate_all()
 
     def truncate_all(self):
-        conn = psycopg2.connect(self.app.config["PG_CONNECT"])
-        cur = conn.cursor()
-        cur.execute("TRUNCATE highlevel_json CASCADE;")
-        cur.execute("TRUNCATE highlevel CASCADE;")
-        cur.execute("TRUNCATE lowlevel CASCADE;")
-        cur.execute("TRUNCATE statistics CASCADE;")
-        cur.execute("TRUNCATE incremental_dumps CASCADE;")
-        conn.commit()
-        cur.close()
-        conn.close()
+        connection = psycopg2.connect(current_app.config['PG_CONNECT'])
```

Comment:
I don't think these tests are meant to be run on production DB. I can explicitly reference `PG_CONNECT_TEST` there in cleanup code, but in other places it would make rebinding pointless. We'll have to check if tests are being run and switch connection strings assuming that `TESTING` config variable is set properly.